### PR TITLE
added mentor Alejandro to Java track

### PR DESCRIPTION
**Pull request by calmcoconut**

### CODE REVIEW

File: mentors/java/a.json [hunk 1]
```diff
@@ -54,5 +54,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I have been working in Java for the last 13 years at a software major. Beside my everyday work, I am also responsible for teaching Java (Core and J2EE) to new joinees."
+  },
+  {
+    "github_username": "calmcoconut",
+    "name": "Alejandro Diaz",
+    "link_text": Blog,
```
Comment (rpottsoh) [inline mentors/java/a.json:61]: Should be "Blog".  Needs to be in quotes.

**Review by rpottsoh — CHANGES_REQUESTED** [review]

File: mentors/java/a.json (hunk 1 above)
Comment (calmcoconut) [reply mentors/java/a.json]: Thank you, silly mistake.

**Review by rpottsoh — APPROVED** [review]

File: mentors/java/a.json [hunk 2]
```diff
@@ -54,5 +54,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I have been working in Java for the last 13 years at a software major. Beside my everyday work, I am also responsible for teaching Java (Core and J2EE) to new joinees."
+  },
+  {
+    "github_username": "calmcoconut",
```
Comment (rpottsoh) [inline mentors/java/a.json:59]: Your GitHub account name starts with `c`.  You should be adding your bio to c.json not a.json.

**Review by rpottsoh — CHANGES_REQUESTED** [review]
Please move your bio to the correct json file.

**Review by rpottsoh — APPROVED** [review]

**Pull request merged**